license_test.go

```diff
@@ -64,6 +64,25 @@ var otherCheck = regexp.MustCompile(`#
 #
 `)
 
+var tplCheck = regexp.MustCompile(`{/*
```

Comment:
this is a regex pattern I tried and can match license header in `.tpl` file, I think current regex forget to escape some special characters.

```
	var tplCheck = regexp.MustCompile(`{\/\*
Licensed to the Apache Software Foundation \(ASF\) under one
or more contributor license agreements\.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership\.  The ASF licenses this file
to you under the Apache License, Version 2\.0 \(the
"License"\); you may not use this file except in compliance
with the License\.  You may obtain a copy of the License at

  http:\/\/www\.apache\.org\/licenses\/LICENSE-2\.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied\.  See the License for the
specific language governing permissions and limitations
under the License\.
\*\/}`)
```